console/console-init/ui/src/modules/iot-device-detail/components/DeviceInfo/DeviceInfo.tsx

```diff
@@ -72,27 +73,14 @@ export const DeviceInfo: React.FC<IDeviceInfoProps> = ({
     credentials
   };
 
-  const prepareMetadataList = () => {
-    const metadataList = [];
-    if (metadetaJson?.default) {
-      metadataList.push({
-        headers: ["Message info parameter", "Type", "Value"],
-        data: metadetaJson?.default
-      });
-    }
-    if (metadetaJson?.ext) {
-      metadataList.push({
-        headers: ["Basic info parameter", "Type", "Value"],
-        data: metadetaJson?.ext
-      });
-    }
-    return metadataList;
-  };
-
   const onToggle = (isEnabled: boolean) => {
     setIsHidden(isEnabled);
   };
 
+  const isMetaDataPresent = () => {
```

Comment:
please use `D` in lower case in MetaData i.e. isMetadataPresent. Follow same for across the app.